### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by morganizwd**

### CODE REVIEW

**Comment by morganizwd** [comment]
[Задания №1](https://trello.com/c/wR9ezMRF/64-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%961)

File: Task1/Task1/Program.cs [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+﻿using System;
+
+namespace Exercise_1_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значения переменных first и second");
+            Console.Write("first = ");
```
Comment (PaShkadun) [inline Task1/Task1/Program.cs:10]: add whiteline between this lines.

File: Task1/Task1/Program.cs [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+            int first = int.Parse(Console.ReadLine());
+            Console.Write("second = ");
```
Comment (PaShkadun) [inline Task1/Task1/Program.cs:12]: add whiteline between this lines.

File: Task1/Task1/Program.cs [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+            int second = int.Parse(Console.ReadLine());
+            first = first + second;
```
Comment (PaShkadun) [inline Task1/Task1/Program.cs:14]: add whiteline between this lines.

File: Task1/Task1/Program.cs [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+            second = first - second;
+            first = first - second;
+            Console.Write($"Переменная first = {first}, переменная second = {second}");
+
+
```
Comment (PaShkadun) [inline Task1/Task1/Program.cs:19]: Remove this useless whitelines.

**Pull request merged**